src/main/java/app/linguistai/bmvp/service/AccountService.java

```diff
@@ -284,4 +284,18 @@ public boolean setPassword(String email, String password) throws Exception {
             throw new SomethingWentWrongException();
         }
     }
+
+    private void initiateUserSession(String email) throws Exception {
+        // Upon successful user entry, check whether to increase user streak or not
+        // ALSO IN MESSAGE SERVICE: userStreakService.updateUserStreak(dbUser.getEmail());
```

Comment:
I carried over this comment line but I am not sure if it is still relevant since message service is deprecated. @can-aslan

Reply:
We can remove the comment here, since the message service is deprecated and will be removed in a future sprint. Thanks for pointing it out